files/BatteryTools.py

```diff
@@ -20,11 +20,20 @@ def battery_model_sizing(model, desired_power, desired_capacity, desired_voltage
         volts
     :param size_by_ac_not_dc: optional bool
         Sizes for power and capacity are on AC side not DC side of battery-inverter regardless of connection type
+    :param module_dict: optional dict
+        Module specs for scaling surface area
+            capacity: float
+                Utility-scale module capacity for scaling surface area
+                Battery: kWhAC if AC-connected, kWhDC otherwise
+                BatteryStateful: battery kWhDC
+            surface_area: float
+                Utility-scale module surface area m^2
+                m^2 of module battery
```

Comment:
I could see this being used outside of a utility-scale as well, so maybe the phrase "Utility-scale module" could just be "Single battery module"